programs/marinade-finance/src/instructions/user/deposit_stake_account.rs

```diff
@@ -104,7 +104,7 @@ impl<'info> DepositStakeAccount<'info> {
             MarinadeError::RequiredActiveStake
         );
 
-        // require the stake is active since current_epoch + WAIT_EPOCHS
+        // require the stake to be active for at least WAIT_EPOCHS
```

Comment:
It is accepting non-active stakes also. activation_epoch means epoch when it was created not epoch when it was activated

Reply:
fixed